hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java

```diff
@@ -334,28 +327,13 @@ private boolean getFilesForArchive(DBCheckpoint checkpoint,
         compactionLogDir.getOriginalDir().toPath());
   }
 
-  private void logEstimatedTarballSize(
-      DBCheckpoint checkpoint, boolean includeSnapshotData) {
-    try {
-      Counters.PathCounters counters = Counters.longPathCounters();
-      CountingPathVisitor visitor = new CountingPathVisitor(
-          counters, SST_FILE_FILTER, TRUE);
-      Files.walkFileTree(checkpoint.getCheckpointLocation(), visitor);
-      int totalSnapshots = 0;
-      if (includeSnapshotData) {
-        Set<Path> snapshotPaths = getSnapshotDirs(checkpoint, false);
-        totalSnapshots = snapshotPaths.size();
-        for (Path snapshotDir: snapshotPaths) {
-          Files.walkFileTree(snapshotDir, visitor);
-        }
-      }
-      LOG.info("Estimates for Checkpoint Tarball Stream - Data size: {} KB, " + "SST files: {}{}",
-          counters.getByteCounter().get() / (1024),
-          counters.getFileCounter().get(),
-          (includeSnapshotData ? ", snapshots: " + totalSnapshots : ""));
-    } catch (Exception e) {
-      LOG.error("Could not estimate size of transfer to Checkpoint Tarball Stream.", e);
+  private void logEstimatedTarballSize(DBCheckpoint checkpoint, boolean includeSnapshotData)
+      throws IOException {
+    Set<Path> snapshotPaths = new HashSet<>();
+    if (includeSnapshotData) {
+      snapshotPaths = getSnapshotDirs(checkpoint, true);
```

Comment:
nit: to get an initial estimated size of transfer we can just get the size of whatever snapshot directories exist on the disk. We need not wait or the snapshot directories to exist.

Reply:
Done.